sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -139,15 +146,19 @@ export default defineComponent({
             draftAlreadyExists: false,
             draftId: id,
           };
-          const programYear =
-            await ProgramYearService.shared.getActiveProgramYear(
-              programYearId.value,
+          const dynamicFormConfiguration =
+            await DynamicFormConfigurationService.shared.getDynamicFormConfiguration(
+              DynamicFormType.StudentFinancialAidApplication,
+              {
+                programYearId: programYearId.value,
+                offeringIntensity: selectedIntensity,
+              },
             );
-          if (programYear) {
+          if (dynamicFormConfiguration) {
```

Comment:
This behavior is not been introduced here, but doing the check would not make the method fail silently?
Maybe we can just add an "unexpected error" snack?